File: data/data.code.javascript.client/AckRoleLoginOk.js
module.exports = AckRoleLoginOk;

var Packet = require('../net/Packet');


function AckRoleLoginOk() {
	this._uname = "";


	this.Decode = function(packet) {
		this._uname = packet.ReadString();
	}
}

AckRoleLoginOk.prototype = {
	set Uname(val) {
		this._uname = val;
	},
	get Uname() {
		return this._uname;
	},
}

File: data/data.code.lua.client/AckRoleLoginOk.lua
module("AckRoleLoginOk", package.seeall)


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = AckRoleLoginOk})
	tab_obj:init()
	return tab_obj
end

function init(self)
	self.uname = ""
end


function decode(self, pack)
	self.uname = pack:readString()
	return self
end

return AckRoleLoginOk
